torch/_inductor/codegen/cpp.py

```diff
@@ -724,6 +728,10 @@ def erf(x):
     def erfc(x):
         return f"std::erfc({x})"
 
+    @staticmethod
+    def erfinv(x):
+        return f"calc_erfinv({x})"
```

Comment:
@kshitij12345 to fix the arm failures, you need to add this line to `torch/_inductor/codegen/cpp_prefix.h`
```cpp
#include <ATen/native/Math.h>
```

It works on x86 because they support vectorization, and the vector headers import `Math.h` for you.

Reply:
Thank you for pointing this out :)